### PR TITLE
Development week2

### PR DESCRIPTION
**Pull request by shijiez777**
Pull request for week 2 development. Implemented data downloading, pdf OCR recognition and storing, and a SVM model with tfidf feature for testing classification result.

### CODE REVIEW

File: Insight_Project_Framework/OCR.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            # tmp_texts = []
+            # for image in images:
+            #     tmp_texts.append(pytesseract.image_to_string(image))
+            # # dump file using pickle
+            text_file_name = os.path.join(text_folder_path, text_name) + '.pkl'        
+            with open(text_file_name, 'wb') as filehandle:
+                pickle.dump(tmp_text, filehandle)
+            print(i, end='\r')
+        # break for testing
+        if i == 1000:
```
Comment (sdliuyuzhi) [inline Insight_Project_Framework/OCR.py:70]: This is fine for developing stage. Ultimately we want to create parameters or separate scripts for testing.

File: Insight_Project_Framework/doc_type_classifier.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+from sklearn import preprocessing
+
+def read_pickle(text_folder_path, file_name):
+
+    with open(os.path.join(text_folder_path, file_name), 'rb') as filehandle:
+        # read the data as binary data stream
+        read_text = pickle.load(filehandle)
+    return read_text
+
+
```
Comment (sdliuyuzhi) [inline Insight_Project_Framework/doc_type_classifier.py:15]: Is this all we need to clean up the text? We can add other functions later on if needed. For now, it's fine.

**Review by sdliuyuzhi — APPROVED** [review]
Looks good to me at this stage.

File: Insight_Project_Framework/OCR.py (hunk 1 above)
Comment (shijiez777) [reply Insight_Project_Framework/OCR.py:70]: Ok. I will create testing functions later.

File: Insight_Project_Framework/doc_type_classifier.py (hunk 2 above)
Comment (shijiez777) [reply Insight_Project_Framework/doc_type_classifier.py:15]: For now I haven't added too much text preprocessing, except for removing new lines and several other line separation expressions. I will modify and fine-tune theses preprocessing steps this week.

**Pull request merged**